apps/docs/src/components/ComponentSignature/ComponentSignature.astro

```diff
@@ -99,10 +98,10 @@ const id = Astro.props.idOverride ?? 'component-signature'
                 <h3 class="mt-0">Props</h3>
 
                 <div
-                  class={c(
+                  class:list={[
```

Comment:
should this just be `class={[`?

Reply:
my understanding is that astro does `class:list={[...]}` where as svelte does `class={[...]}` but they might've changed since last i looked 🤔

Reply:
ah! thanks for the docs link